src/libs/actions/IOU.js

```diff
@@ -49,34 +49,48 @@ Onyx.connect({
  * @param {Object} report
  * @param {Number} amount - always in the smallest unit of the currency
  * @param {String} currency
- * @param {String} recipientEmail
+ * @param {String} payeeEmail
  * @param {Object} participant
  * @param {String} comment
  */
-function requestMoney(report, amount, currency, recipientEmail, participant, comment) {
-    const debtorEmail = OptionsListUtils.addSMSDomainIfPhoneNumber(participant.login);
+function requestMoney(report, amount, currency, payeeEmail, participant, comment) {
+    const payerEmail = OptionsListUtils.addSMSDomainIfPhoneNumber(participant.login);
     let chatReport = lodashGet(report, 'reportID', null) ? report : null;
+    const isPolicyExpenseChat = participant.isPolicyExpenseChat || participant.isOwnPolicyExpenseChat;
```

Comment:
```suggestion
    const isPolicyExpenseChat = report.isPolicyExpenseChat || report.isOwnPolicyExpenseChat;
```

Reply:
This won't work if requesting from global menu (no report).

Reply:
Correct, we need to use the selected participant instead of the report because as @s77rt pointed out, there's no report when we use the global create.